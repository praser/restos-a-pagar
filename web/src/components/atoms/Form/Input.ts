import styled from 'styled-components';

const Input = styled.input`
  display: block;
  padding: 0.375rem 0.75rem;
  font-size: 1rem;
  font-weight: 400;
  line-height: 1.5;
  color: #6e707e;
  background-color: #fff;
  background-clip: padding-box;
  border: 1px solid #d1d3e2;
  border-radius: 0.35rem;

  &:focus {
    color: #6e707e;
    background-color: #fff;
    border-color: #bac8f3;
    outline: 0;
    box-shadow: 0 0 0 0.2rem rgba(78, 115, 223, 0.25);
  }
`;

export default Input;
