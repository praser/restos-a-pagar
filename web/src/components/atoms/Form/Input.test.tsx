import React from 'react';

import renderer from 'react-test-renderer';

import Input from './Input';

describe('Input atom component', () => {
  it('is expected to render the component correctly', () => {
    const input = renderer.create(<Input />);
    expect(input).toMatchSnapshot();
  });
});
