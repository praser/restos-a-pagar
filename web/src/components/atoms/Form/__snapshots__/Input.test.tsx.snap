// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input atom component is expected to render the component correctly 1`] = `
<input
  className="sc-bdfBwQ iXezPU"
/>
`;
